src/Footer/Footer.js

```diff
@@ -0,0 +1,40 @@
+import React from "react";
+
+import "./Footer.scss";
+import netiflyDark from "./images/netiflyDark.svg";
+
+class Footer extends React.Component {
+  render() {
+    return (
+      <div className="footer">
+        <div className="footer-content">
+          <div className="footer-top">
+            <span className="footer-links">
+              <a href="/terms" className="link link-footer">
+                Terms of Service
+              </a>
+              <a href="/privacy" className="link link-footer">
+                Privacy Policy
+              </a>
+              <a href="/credits" className="link link-footer">
+                Credits
+              </a>
+              <a href="/about" className="link link-footer">
+                About
+              </a>
+            </span>
+            <span className="copyright">Copyright © 2015 Recipe Shelf</span>
```

Comment:
It might be good to add some js to calculate the current year, instead of hardcoding it